src/docs/product/alerts-notifications/troubleshooting.mdx

```diff
@@ -0,0 +1,25 @@
+---
+title: Troubleshooting
```

Comment:
I think we can get rid of this page and put the information into other pages... if organized correctly we shouldn't need it

Reply:
Many sections of Docs (for example, the SDKs) have a Troubleshooting section now and keeping this Troubleshooting page will add consistency throughout docs.